salt/utils/win_reg.py

```diff
@@ -767,6 +767,9 @@ def cast_vdata(vdata=None, vtype='REG_SZ'):
     # Make sure REG_MULTI_SZ is a list of strings
     elif vtype_value == win32con.REG_MULTI_SZ:
         return [_to_unicode(i) for i in vdata]
+    # Make sure REG_QWORD is a 64 bit integer
+    elif vtype_value == win32con.REG_QWORD:
+        return vdata if six.PY3 else long(vdata)  # pylint: disable=W1699
```

Comment:
What about using `six.integer_types` here?

Seems nicer that disabling the lint error.

https://pythonhosted.org/six/#six.integer_types

Reply:
I tried that, but it seems to only be useful for detecting types not casting.
```
>>> from salt.ext import six
>>> long(7)
7L
>>> six.integer_types(7)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'tuple' object is not callable
>>> isinstance(7, six.integer_types)
True
>>> isinstance(7L, six.integer_types)
True
>>>
```

Reply:
@rallytime You want I should add `future` as a dependency instead? That works and is quite clean.

Reply:
Ah, right, OK. I think this is fine as-is. I'd rather not add the dependency. Your current check works.